pallets/oracle/src/tests.rs

```diff
@@ -205,7 +205,7 @@ fn test_is_invalidated() {
 #[test]
 fn set_xlm_tx_fees_per_byte_succeeds() {
```

Comment:
I guess we can remove this test altogether, since by removing the FeeEstimation item, we remove the need for the test

Reply:
Done.